backend/app/api/routes.py

```diff
@@ -474,7 +474,10 @@ def send_initialized_data_to_db(repo_info, code_files):
                 upsert=True
             )
 
+        repo_embeddings = db.get_repo_files_embeddings(repo_id)
```

Comment:
This function call should be moved to the /report endpoint after embeddings are recomputed (i.e., line 126)

Where it is currently may become outdated if SHAs are different between bug reports.